.appveyor.yml

```diff
@@ -12,26 +12,30 @@ install:
     # The AppVeyor 'rollout builds' option is supposed to serve the same
     # purpose but it is problematic because it tends to cancel builds pushed
     # directly to master instead of just PR builds (or the converse).
+    # credits: JuliaLang developers.
     - ps: if ($env:APPVEYOR_PULL_REQUEST_NUMBER -and $env:APPVEYOR_BUILD_NUMBER -ne ((Invoke-RestMethod `
          https://ci.appveyor.com/api/projects/$env:APPVEYOR_ACCOUNT_NAME/$env:APPVEYOR_PROJECT_SLUG/history?recordsNumber=50).builds | `
          Where-Object pullRequestId -eq $env:APPVEYOR_PULL_REQUEST_NUMBER)[0].buildNumber) { `
            throw "There are newer queued builds for this pull request, failing early." }
 
     # Add path, activate `conda` and update conda.
-    - cmd: set "PATH=%CONDA_INSTALL_LOCN%\\Scripts;%CONDA_INSTALL_LOCN%\\Library\\bin;%PATH%"
     - cmd: call %CONDA_INSTALL_LOCN%\Scripts\activate.bat
-    - cmd: conda config --set always_yes yes --set changeps1 no --set show_channel_urls true
-    - cmd: conda update conda
-    - cmd: conda config --add channels conda-forge --force
+    - cmd: conda.exe config --set always_yes yes --set changeps1 no --set show_channel_urls true
+    - cmd: conda.exe update conda
+    - cmd: conda.exe config --add channels conda-forge --force
 
     - cmd: set PYTHONUNBUFFERED=1
 
-    - cmd: conda install conda-build
-    - cmd: conda info --all
-    - cmd: conda list
+    - cmd: conda.exe info --all
+    - cmd: conda.exe list
+
 
 # Skip .NET project specific build phase.
 build: off
 
 test_script:
-    - "conda build conda.recipe"
+    - conda.exe create --name TEST python=%PY% configparser docopt flask jinja2 lxml python-dateutil pytz spyne pytest odm2api requests sqlalchemy suds-jurko
+    - conda activate TEST
+    - python -m pip install . --no-deps -vv
+    - python wof/examples/flask/cbi/build_cbi_cache.py || exit 1
+    - pytest -s -rxs -v -k "not test_odm2_dao_sqlite"
```

Comment:
@emiliom we exchanged 1-line here and all the lines in the `conda.recipe` for these 5 lines, shorter and easier to follow b/c now the tests are all in a single file.